scripts/feature_tests/pivoting/Makefile

```diff
@@ -15,3 +15,6 @@ repivoting:
 
 deprovision:
 	./../feature_test_deprovisioning.sh
+
+upgrading:
```

Comment:
Who is calling this Makefile target?

Reply:
This will be called by the scrip that Anwar is working on a priority test case and he will pivot with make upgrading with 3cp +1 worker.

Reply:
As an alternative, I just realized that wouldn't it be possible to rely on `pivoting` target instead of creating a new one? If case of upgrade test you would need to export NUM_OF_MASTER_REPLICAS & NUM_OF_WORKER_REPLICAS with desired numbers before you call the `make pivoting`. Because, at the end of the day both pivoting and upgrading are calling pivot.yml

Also, I think it would be better use imperative verb for the new target, i.e. `upgrade`, for the sake of the consistency.

Reply:
@fmuyassarov pivoting Makefile exports those variables too, as discussed yesterday, it would need an script after this to override the variables I believe

Reply:
It might be that I don't see the whole flow here but regarding the 3cp+1w priority test case, the NUM_OF_ variables are set to correct values as a first thing when the test case is started. Control plane and worker replicas are provisioned after that.

Reply:
> @fmuyassarov pivoting Makefile exports those variables too, as discussed yesterday, it would need an script after this to override the variables I believe

Ops, sorry I missed that part. I'm ok with the current state. But another option would be to not export those vars in pivoting Makefile since pivoting scripts run with 2 nodes (default value in Ansible ..../vars.yaml). That would avoid the need to override the vars in the Makefile. WDYT @kashifest ?

Reply:
If we are calling the` make pivoting` ,  it calls verify.yml and in our upgrade case it not needed. Also make file for pivoting its declare those variable with 1 for CP and 1 for Worker but we need 3 for cp and 1 for worker in the move.yml file. That is the reason we use separate script for upgrade task.

Reply:
@fmuyassarov that can be an option, we need to test it

Reply:
> upgrade has different use-cases and different number of CP and worker anyhow,

Can't they export the vars with different values every time they run different upgrade tests?

Reply:
yeah true but still for pivoting then we need to export it separately if not default value if we dont do it here.

Reply:
well actually we can take the export in the pivot.sh

Reply:
Also to be noted to speed up our test after pivoting, we prioritize 3cp+1w use case.